tcms/telemetry/api.py

```diff
@@ -117,6 +117,9 @@ def status_matrix(query=None):
     }
 
 
+_TOTAL = str(_('TOTAL'))
```

Comment:
Don't use a constant, use `str(_('TOTAL'))` inside the functions bodies below b/c this doesn't render as translated.

I think the reason may be that actual string value is calculated very early before the translation machinery (and maybe most of Django) is loaded. 

If you insist on having a module level variable, then move it to the top and cast to str whenever this is used, not on the top-level.  This will also resolve the JS display issues.